.github/workflows/verifyaotcompat.yml

```diff
@@ -10,12 +10,15 @@ jobs:
       fail-fast: false # ensures the entire test matrix is run, even if one permutation fails
       matrix:
         os: [ ubuntu-latest, windows-latest ]
-        version: [ net8.0 ]
+        version: [ net8.0, net9.0 ]
 
     runs-on: ${{ matrix.os }}
     steps:
     - uses: actions/checkout@v4
 
+    - name: Setup dotnet  # The runner does not have .NET 9 SDK installed by default yet.
```

Comment:
```suggestion
    - name: Setup dotnet
```

Reply:
@xiang17 FYI I removed the comment. Most of the workflows have this step I think it is OK just to have it there forever and call it a standard thing 😄